opencxl/cxl/component/cache_controller.py

```diff
@@ -183,6 +183,7 @@ async def _memory_load(self, address: int, size: int) -> CacheResponse:
     async def _memory_store(self, address: int, size: int, value: int) -> None:
         packet = CacheRequest(CACHE_REQUEST_TYPE.WRITE_BACK, address, size, value)
         await self._cache_to_coh_agent_fifo.request.put(packet)
+        packet = await self._cache_to_coh_agent_fifo.response.get()
```

Comment:
Nothing is being done with `packet`. Is this intentional?

Reply:
Yes on purpose. I removed the "packet =" part.